WordPress/Classes/ViewRelated/NUX/WordPressAuthenticationManager.swift

```diff
@@ -79,9 +79,9 @@ class WordPressAuthenticationManager: NSObject {
                                                               textButtonColor: .primary,
                                                               textButtonHighlightColor: .primaryDark,
                                                               viewControllerBackgroundColor: .basicBackground,
-                                                              navBarBackgroundColor: .basicBackground,
-                                                              navButtonTextColor: .brand,
-                                                              navTitleTextColor: .text)
+                                                              navBarBackgroundColor: .basicBackground,  // TODO: Replace with .appBarBackground once new nav bar styles are merged
```

Comment:
can we add a feature-specific keyword to these TODOs so they are easily searchable when it's time to remove them? Like TODO - BIGTITLES